index.html

```diff
@@ -3,40 +3,42 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>판다마켓</title>
-    <link
-      rel="stylesheet"
-      as="style"
-      crossorigin
-      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable.min.css"
+    <meta property="og: title" content="판다마켓" />
+    <meta
+      property="og: url"
+      content="https://pappaya-pandamarket.netlify.app/"
+    />
+    <meta
+      property="og: description"
+      content="일상의 모든 물건을 거래해보세요."
     />
+    <meta property="og: image" content="../img/opengraph_image" />
+    <title>판다마켓</title>
     <link rel="stylesheet" href="./styles/reset.css" />
     <link rel="stylesheet" href="./styles/style.css" />
   </head>
   <body>
-    <header>
+    <header id="header">
       <nav>
-        <a href="/">
-          <img src="./img/logo.png" alt="logo" />
-        </a>
+        <a href="/"><div class="logo_img"></div> </a>
```

Comment:
❗️ 수정요청
지금 디자인에서 괜찮을 것 같지만 공백은 의도치 않은 동작을 발생시킬 수 있습니다~ 
필요한 공백이 아니면 가능하면 습관적처럼 없애는 것을 추천드려요~

```suggestion
        <a href="/"><div class="logo_img"></div></a>
```